### PR TITLE
Update Q20

### PR DESCRIPTION
**Pull request by j-j-gajjar**

### CODE REVIEW

File: nosql/nosql-quiz.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Q19. You want to test HBase on your laptop using the fewest number of steps. Wha
 Q20. You need to design security for DynamoDB to allow users read-obly access to certain items and attriutes in a table. What do you do?
 
     - Use IAM roles.
-    - Use IAM plicy confditions. <<<<----Correct
+    - Use IAM plicy conditions. <<<<----Correct
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:59]: @j-j-gajjar you might want to update this one as well
```suggestion
    - Use IAM policy conditions. <<<<----Correct
```
thank you in advance!

**Review by Ebazhanov — CHANGES_REQUESTED** [review]

**Review by Ebazhanov — APPROVED** [review]
Thank you for the improvements @j-j-gajjar, looks very good! :+1:

**Comment by Ebazhanov** [comment]
@all-contributors please add @j-j-gajjar for design

**Comment by j-j-gajjar** [comment]
Thank You @Ebazhanov

**Pull request merged**